Defs/Ammo/Pistols/9x19mmPara_NL.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
```

Comment:
As discussed, tranq ammo should be patched conditionally.

Reply:
Temporarily removed as part of 0a721e3, and will be patched conditionally at a later date